### PR TITLE
[Viva Learning]Documentation PR for V1 of Catalog Sync Graph Api's

### PR DESCRIPTION
**Pull request by SritejaKampara**
Documentation PR for V1 of Catalog Sync Graph Api's for Learning Content and Learning Provider.

### CODE REVIEW

File: api-reference/v1.0/resources/learningprovider.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[List learningProviders](../api/employeeexperience-list-learningproviders.md)|[learningProvider](../resources/learningprovider.md) collection|Get a list of the [learningProvider](../resources/learningprovider.md) objects and their properties.|
+|[Create learningProvider](../api/employeeexperience-post-learningproviders.md)|[learningProvider](../resources/learningprovider.md)|Create a new [learningProvider](../resources/learningprovider.md) object.|
+|[Get learningProvider](../api/learningprovider-get.md)|[learningProvider](../resources/learningprovider.md)|Read the properties and relationships of a [learningProvider](../resources/learningprovider.md) object.|
+|[Update learningProvider](../api/learningprovider-update.md)|[learningProvider](../resources/learningprovider.md)|Update the properties of a [learningProvider](../resources/learningprovider.md) object.|
+|[Delete learningProvider](../api/employeeexperience-delete-learningproviders.md)|None|Deletes a [learningProvider](../resources/learningprovider.md) object.|
+
+## Properties
```
[the remark this answers is not in the document]
Comment (Jarbas-MSFT) [reply api-reference/v1.0/resources/learningprovider.md:28]: Property was removed from BETA.

File: api-reference/v1.0/resources/employeeexperience.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Namespace: microsoft.graph
+
+
+Represents a container that exposes navigation properties for employee experience resources.
+
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[Get employeeExperience](../api/employeeexperience-get.md)|[employeeExperience](../resources/employeeexperience.md)|Read the properties and relationships of an [employeeExperience](../resources/employeeexperience.md) object.|
```
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/resources/employeeexperience.md]: Removed

File: api-reference/v1.0/resources/learningprovider.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/resources/learningprovider.md:28]: Yes your understanding is correct.

File: api-reference/v1.0/api/learningcontent-get.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+The following example shows the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "get_learningcontent",
+  "sampleKeys": ["13727311-e7bb-470d-8b20-6a23d9030d70", "LP4471"]
+}
+-->
+``` http
+GET https://graph.microsoft.com/v1.0/employeeExperience/learningProviders/13727311-e7bb-470d-8b20-6a23d9030d70/learningContents(externalId='LP4471')
```
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/api/learningcontent-get.md]: Fixed at both places as suggested.

File: api-reference/v1.0/api/learningcontent-get.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+The following example shows a request that retrieves the metadata of a learning content based on its ID (primary key).
+
+#### Request
+The following example shows the request.
+
+<!-- {
+  "blockType": "request",
+  "name": "get_learningcontent",
+  "sampleKeys": ["13727311-e7bb-470d-8b20-6a23d9030d70", "LP4471"]
```
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/api/learningcontent-get.md]: Removed

File: api-reference/v1.0/resources/learningcontent.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+Represents the metadata of content for employee learning.
+
+
+Inherits from [entity](../resources/entity.md).
+
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[List learningContents](../api/learningprovider-list-learningcontents.md)|[learningContent](../resources/learningcontent.md) collection|Get a list of the [learningContent](../resources/learningcontent.md) objects and their properties.|
+|[Create learningContent](../api/learningprovider-post-learningcontents.md)|[learningContent](../resources/learningcontent.md)|Create a new [learningContent](../resources/learningcontent.md) object.|
```
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/resources/learningcontent.md]: Removed

File: api-reference/v1.0/api/employeeexperience-delete-learningproviders.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+|Application|Not supported.|
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+DELETE /employeeExperience/learningProviders/{learningProviderId}
```
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/api/employeeexperience-delete-learningproviders.md]: Updated accordingly

File: api-reference/v1.0/api/learningprovider-delete-learningcontents.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+For a specific learning content based on its ID (primary key):
+``` http
+DELETE /employeeExperience/learningProviders/{learningProviderId}/learningContents/{learningContentId}
```
[the remark this answers is not in the document]
Comment (SritejaKampara) [reply api-reference/v1.0/api/learningprovider-delete-learningcontents.md]: Updated accordingly

**Comment by Jarbas-MSFT** [comment]
@Lauragra this PR is ready for final review.

I fixed issues also in related BETA topics to keep the doc set consistent.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**